.github/workflows/publish-wiki.yml

```diff
@@ -0,0 +1,23 @@
+---
+name: Publish wiki
+
+on:
+  push:
+    branches: [main]
+    paths:
+      - wiki/**
+      - .github/workflows/publish-wiki.yml
+
+concurrency:
+  group: publish-wiki
+  cancel-in-progress: true
```

Comment:
Is this to avoid race condition when multiple PRs are merged at the same time related to `wiki` folder? Also where is the group defined? Is `group == job name`?

Reply:
I think it is supposed to avoid race conditions for this workflow like you described. The concurrency docs are [here](https://docs.github.com/en/actions/using-jobs/using-concurrency)